pegasus/helpers/section_api_helpers.rb

```diff
@@ -297,7 +297,7 @@ def self.create(params)
       )
     rescue Sequel::UniqueConstraintViolation
       tries += 1
-      retry if tries < 2
+      retry if tries < 3
```

Comment:
Is it appropriate to change [`SectionHelpers#random_code`](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/section_helpers.rb#L7-L9) to generate codes until the code is unique?